src/index.js

```diff
@@ -0,0 +1,4 @@
+import React from  'react';
+import ReactDom from 'react-dom';
+import App from './containers/app';
+ReactDom.render(<App/>, document.getElementById('App'));
```

Comment:
Hey @anyatibrian, great work. Kindly add final new lines at this file and all the other files.

Reply:
which files@Genza cos those other files are just sample files to guide you

Reply:
Hey @anyatibrian , src/components/app.js, babelrc, store.js, webpack.config.js

Reply:
yah @Genza999 its always good to follow the ASK format since it prevents you from giving abstract feedback.